src/testdrive/src/action.rs

```diff
@@ -85,11 +85,17 @@ pub struct Config {
     pub default_timeout: Duration,
     /// A random number to distinguish each run of a testdrive script.
     pub seed: Option<u32>,
+    /// Force the use of a specific temporary directory
+    pub temp_dir: Option<String>,
 }
 
+// We never read tempfile_handle as we only need it so that the tempfile is destroyed
+// when going out of scope. Rustc however warns about dead code
+#[allow(dead_code)]
```

Comment:
Rather than allowing `dead_code` on the whole struct, convention is to prefix the one member that is written but never read with an `_`...

Reply:
Thanks! Since I am doing strictly compiler-driven development and the compiler did not offer me the underscore trick, I thought it applies only to unused variables within functions :-)